README.md

```diff
@@ -33,14 +32,29 @@ require 'surro-gate'
 proxy = SurroGate.new
 
 # Create two TCP socket connections
```

Comment:
2 sockets, one for each direction

Reply:
no, 2 sockets to proxy between them

Reply:
aah cool. Could you change comment for this?